docs/devs/Optimization-Tips.md

```diff
@@ -55,10 +55,27 @@ Add Javascript code like docs/api/Profiler.md, Iot.js will generate a Profiler f
 
 Users can generate collapse stack trace file.
 ```shell
-tools/profiler/cl.py Profiler-123 Profiler-123.dump > col.txt
+deps/jerry/tools/profiler/cl.py Profiler-123 Profiler-123.dump > col.txt
 ```
 
 Then generate flame graph html file.
 ```shell
-tools/profiler/flamegraph.pl col.txt > prof.html
+deps/jerry/tools/profiler/flamegraph.pl col.txt > prof.html
 ```
+
+## Heap Profiler
+Adding below arguments when building and running IoT.js will enable Heap Profiler.
+
+```shell
+$ ./tools/build.py --jerry-heap-profiler
+```
+
+Add Javascript code like docs/api/Profiler.md, Iot.js will generate a Jerry Heap Profiler file, say Profiler-123.
```

Comment:
Javascript => JavaScript, and it's better that the docs/api/Profiler.md would be linkable :)